README.md

```diff
@@ -286,10 +286,21 @@ DEBUG level logging enables logging of:
 
 **Note:** Debug level logging may expose sensitive data in the logs
 
+## Security
+This library does not maintain or manage authentication tokens. The consumer of the SDK is responsible for writing code to retrieve, store, and refresh authentication tokens.
+
+ This library limits its runtime dependencies to reduce the total cost of ownership as much as possible. However, we recommend that you have security stakeholders review all third-party products (3PP) and their dependencies.
+
+For more information about security considerations related to developing headless commerce applications, see 
+[Security Considerations for Headless Commerce](https://developer.commercecloud.com/s/article/HeadlessSecurity) on the 
+[Commerce Cloud Developer Center](https://developer.commercecloud.com).
+
+If you discover any potential security issues, please report them to security@salesforce.com as soon as possible.
+
+
 ## Additional Documentation
 
-- [Examples](./examples)
```

Comment:
Looks like this commit accidentally removes the Examples link that was recently added.

Reply:
Oh yes, good catch!

Reply:
please add relative path to the examples (./examples)

Reply:
> please add relative path to the examples (./examples)

Actually, please provide absolute paths so they won't be broken when deployed to NPM.  But you don't have to fix that. We should.